******************************************
Starting ApplyFinalBDT 
Sun Apr 28 20:30:51 EDT 2019
WD = /data1/avenkate/JpsiLambda_RESTART
******************************************
******************************************
Processing Run 1 LL Data0
******************************************
******************************************
Input file = rootFiles/dataFiles/JpsiLambda/run1/jpsilambda_cutoutks_LL_nonZeroTracks_noPID.root
Output file = rootFiles/dataFiles/JpsiLambda/run1/jpsilambda_LL_FinalBDT2_iso1_v1_noPID.root
******************************************
Incoming Entries = 3632803<FATAL>                          : <BookMVA> fatal error: unable to open input weight file: dataset/weights/TMVAClassification-JpsiLambdaLL_dataRun1_iso1_v1_BDT2_noPID_BDTconf2.weights.xml
***> abort program execution
Error in <TRint::HandleTermInput()>: std::runtime_error caught: FATAL error
[0mroot [1] Error in <RootX11IOErrorHandler>: fatal X11 error (connection to server lost?!)

**** Save data and exit application ****


 *** Break *** segmentation violation
[0m


===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00000031d28ac89e in waitpid () from /lib64/libc.so.6
#1  0x00000031d283e4e9 in do_system () from /lib64/libc.so.6
#2  0x00007fd398626568 in TUnixSystem::StackTrace() () from /cvmfs/lhcb.cern.ch/lib/lcg/releases/ROOT/6.12.06-0f687/x86_64-slc6-gcc62-opt/lib/libCore.so
#3  0x00007fd398628a6c in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/lhcb.cern.ch/lib/lcg/releases/ROOT/6.12.06-0f687/x86_64-slc6-gcc62-opt/lib/libCore.so
#4  <signal handler called>
#5  0x00007fd398558370 in TSysEvtHandler::Activate() () from /cvmfs/lhcb.cern.ch/lib/lcg/releases/ROOT/6.12.06-0f687/x86_64-slc6-gcc62-opt/lib/libCore.so
#6  0x00007fd3989b8daa in TRint::HandleTermInput() () from /cvmfs/lhcb.cern.ch/lib/lcg/releases/ROOT/6.12.06-0f687/x86_64-slc6-gcc62-opt/lib/libRint.so
#7  0x0000000000000004 in ?? ()
#8  0x00007fd3985596a0 in ?? () from /cvmfs/lhcb.cern.ch/lib/lcg/releases/ROOT/6.12.06-0f687/x86_64-slc6-gcc62-opt/lib/libCore.so
#9  0x0000000000000000 in ?? ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum.
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs. Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x00007fd398558370 in TSysEvtHandler::Activate() () from /cvmfs/lhcb.cern.ch/lib/lcg/releases/ROOT/6.12.06-0f687/x86_64-slc6-gcc62-opt/lib/libCore.so
===========================================================


[0mRoot > [0m